Orange/widgets/data/owdatasets.py

```diff
@@ -522,15 +528,17 @@ def onDeleteWidget(self):
             self.__awaiting_state.pb.advance.disconnect(self.__progress_advance)
             self.__awaiting_state = None
 
-    def sizeHint(self):
+    @staticmethod
+    def sizeHint():
         return QSize(900, 600)
 
     def closeEvent(self, event):
         self.splitter_state = bytes(self.splitter.saveState())
         self.header_state = bytes(self.view.header().saveState())
         super().closeEvent(event)
 
-    def load_data(self, path):
+    @staticmethod
+    def load_data(path):
```

Comment:
This method does not do much here, but is overridden (at least) in orange3-single-cell. It is not a staticmethod there, so we might want to keep it non-static here as well for consistency.

Reply:
Agreed.